content/python/concepts/built-in-functions/terms/dir/dir.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'dir()'
+Description: 'Returns the list of valid attributes of the passed object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Objects'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The **`dir()`** function returns the list of valid attributes of the passed object.
+
+## Syntax
+
+```pseudo
+dir(object)
+```
+
+## Parameters
```

Comment:
Remove this header and the following blank line so the sentence below is in the syntax section.

Reply:
@SSwiniarski  I modified it. Could you check it?